src/mlagility/analysis/util.py

```diff
@@ -70,13 +70,20 @@ def clean_output_dir(output_dir: str = filesystem.DEFAULT_CACHE_DIR) -> None:
     """
     output_dir = os.path.expanduser(output_dir)
     ext_list = [".txt", ".out", ".yaml", ".json"]
+    benchmarking_folders = [
```

Comment:
Use mlagility.api.devices.BenchmarkPaths instead of hardcoding this.

https://github.com/groq/mlagility/blob/a9cb737c880ddd6fb2938b2f8ca066f525e9ecea/src/mlagility/api/devices.py#L13

Reply:
Completely changed the approach for something much cleaner. Let me know what you think.